plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestSqlServerConnectorTest.java

```diff
@@ -195,4 +214,45 @@ public static Object[][] timestampTypes()
                 {"timestamp(12)"}
         };
     }
+
+    //TODO replace TableNameDataProvider and ColumnNameDataProvider with ObjectNameDataProvider
```

Comment:
```suggestion
    // TODO replace TableNameDataProvider and ColumnNameDataProvider with ObjectNameDataProvider
```

Reply:
you can intend them with one more space (`//  blah ...`) and then they will

Reply:
cool, didn't know